src/main/webapp/src/app/user-profile/user-profile.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+
+import {UserProfileComponent} from './user-profile.component';
+
+describe('UserProfileComponent', () => {
+  let component: UserProfileComponent;
+  let fixture: ComponentFixture<UserProfileComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [UserProfileComponent],
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(UserProfileComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
Let's add another test case confirming that the content is as expected from mock data.

Reply:
Definitely can do this, but not in this PR. It'll take a little bit of engineering time due to setting up the mock HTTP client though. I've started to populate the Testing section of our design doc with tests that need to be written.